stan/math/opencl/rev/triangular_transpose.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef STAN_MATH_OPENCL_REV_TRIANGULAR_TRANSPOSE_HPP
+#define STAN_MATH_OPENCL_REV_TRIANGULAR_TRANSPOSE_HPP
+#ifdef STAN_OPENCL
+
+#include <stan/math/opencl/matrix_cl_view.hpp>
+#include <stan/math/opencl/triangular_transpose.hpp>
+#include <stan/math/opencl/rev/matrix_cl.hpp>
+
+namespace stan {
+namespace math {
```

Comment:
Just delete this file. We can add later if triangular_transpose is needed.

Reply:
I actually just added it so that that API is the same. Pure lazyness to not add it on my part